### DISCUSSION TITLE
Bug in explore section

### DISCUSSION THREAD

**Discussion by jakbin** [community/community#7991] [Mobile]
When I am in explore section , i little scroll down , then I open a repo , then i go back , it's send me on the top always. (Sorry for my poor english)

**Comment by mxie** [comment]
Hi @jakbin - is this issue still happening? And if so, can you tell us if you're on Android or iOS as well as what app version you're using? Thanks!

**Reply by jakbin** [reply]
Yes , this issue is still. I am using android .

v1.45.0-beta

**Reply by mxie** [reply]
Thanks! One more question: does this happen on both the `For You` section **and** `Trending`? Or just one of these?

**Reply by jakbin** [reply]
Only trending section.

**Reply by mxie** [reply]
Noted. I'll relay this feedback back to the team.
